contracts/farm-bot.sol

```diff
@@ -46,22 +49,29 @@ contract FarmBot is Owned, FarmbotERC20 {
     constructor(
         address _owner,
         address _stakingRewards,
+	address _stakingToken,
         address _revoBounty,
         address _router,
-        address[] memory _path0,
-        address[] memory _path1,
+	address[] memory _rewardsTokens,
+	address[][2][] memory _paths,
         string memory _symbol
     ) Owned(_owner) {
-        stakingRewards = StakingRewards(_stakingRewards);
-        rewardsToken = stakingRewards.rewardsToken();
+        stakingRewards = IMoolaStakingRewards(_stakingRewards);
+
+	for (uint i=0; i<_rewardsTokens.length; i++) {
+	    rewardsTokens.push(IERC20(_rewardsTokens[i]));
+	}
+
+	assert(_paths.length == _rewardsTokens.length);
+	paths = _paths;
+
         revoBounty = IRevoBounty(_revoBounty);
 
-        stakingToken = IUniswapV2Pair(address(stakingRewards.stakingToken()));
+        stakingToken = IUniswapV2Pair(_stakingToken);
```

Comment:
We have to pass this in manually now rather than getting it from the `stakingRewards` contract, since the `IMoolaStakingRewards` interface does not publicly expose the `stakingToken` address. The actual _implementation_ of `MoolaStakingRewards` _does_ expose it publicly, but the interface does... We could just update the interface to expose the `stakingToken` address, but that feels a little weird to me.